app/enssubdomain/views.py

```diff
@@ -79,22 +79,22 @@ def handle_subdomain_exists(request, github_handle):
             'ens_domain': settings.ENS_TLD,
         }
         return TemplateResponse(request, 'ens/ens_pending.html', params)
-    elif request_reset_time > last_request.created_on:
```

Comment:
whats the reasoning behind this change?  seems the same logically to me

Reply:
It is fairly similar.  It's removing redundant if/elif/else logic since there are returns present.  It's a general flake8 linting guideline.
If you want me to split it off and push it to a separate PR for code cleanup, I can do that.